### DISCUSSION TITLE
Training TSTPlus with multiple GPUs

### DISCUSSION THREAD

**Discussion by psinthong** [timeseriesAI/tsai#364] [Q&A]
Hello,

Thank you for providing an impressive library. I'm new to deep learning and I find this library very easy to use. 

I have a couple of questions about training a model on multiple GPUs. Would it be possible to train a TSTPlus model (with MVP) using multiple GPUs? If so, could you point me to an example?

Thanks in advance for your help!

**Comment by oguiza** [comment]
Hi @psinthong,
I haven't tried tsai in a multi-GPU environment (I don't have access to any). 
What I know is that fastai is supposed to work on multi-GPU (https://docs.fast.ai/distributed.html). There's an example demonstrating how to do it (https://github.com/fastai/fastai/blob/master/nbs/examples/distrib.py). Although there's an open issue (https://github.com/fastai/fastai/issues/3498). So I'm not sure if it'll work. It'd be good if you can test it and let me know how it goes. I'll try to help if I can.
